### PR TITLE
add figuresClasses solution

### PR DESCRIPTION
**Pull request by DidenkoDenis306**
One test fails because it asks for a circle area of 113.09 and gets 113.1. toFixed(2) does not work.

### CODE REVIEW

File: src/figuresClasses.ts [hunk 1]
```diff
@@ -1,19 +1,100 @@
...
+
+    this.color = color;
+    this.sides = [a, b, c];
+  }
+
+  getArea(): number {
+    const [a, b, c] = this.sides;
+    const s = (a + b + c) / 2;
+
+    return Number((Math.sqrt(s * (s - a) * (s - b) * (s - c))).toFixed(2));
```
Comment (33Orange) [inline src/figuresClasses.ts:43]: Try to use `Math.floor`

File: src/figuresClasses.ts [hunk 2]
```diff
@@ -1,19 +1,100 @@
...
+    if (radius <= 0) {
+      throw new Error('Radius should be greater than 0');
+    }
+
+    this.color = color;
+    this.radius = radius;
+  }
+
+  getArea(): number {
+    return Number((Math.PI * (this.radius ** 2)).toFixed(2));
```
Comment (33Orange) [inline src/figuresClasses.ts:64]: Try to use `Math.floor`

File: src/figuresClasses.ts [hunk 3]
```diff
@@ -1,19 +1,100 @@
...
+      throw new Error('Width and height should be greater than 0');
+    }
+
+    this.color = color;
+    this.width = width;
+    this.height = height;
+  }
 
+  getArea(): number {
+    return Number((this.width * this.height).toFixed(2));
```
Comment (33Orange) [inline src/figuresClasses.ts:92]: Try to use `Math.floor`

**Review by 33Orange — CHANGES_REQUESTED** [review]
Well done !

A few moments to fix

**Review by 33Orange — CHANGES_REQUESTED** [review]
Well done !

A few moments to fix

**Review by dinaminahi — APPROVED** [review]

**Pull request closed without merging**